core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/stores/dot-experiments-store.service.ts

```diff
@@ -1,21 +1,175 @@
 import { Injectable } from '@angular/core';
-import { ComponentStore } from '@ngrx/component-store';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
 import { LoadingState } from '@portlets/shared/models/shared-models';
+import {
+    DotExperiment,
+    DotExperimentStatusList,
+    GroupedExperimentByStatus,
+    RenderedPageExperiments
+} from '../models/dot-experiments.model';
+import { DotExperimentsService } from '@portlets/dot-experiments/shared/services/dot-experiments.service';
+import { catchError, switchMap, tap, withLatestFrom } from 'rxjs/operators';
+import { HttpErrorResponse } from '@angular/common/http';
+import { EMPTY, Observable, throwError } from 'rxjs';
 
 export interface DotExperimentsState {
     pageId: string;
-    state: LoadingState;
+    pageTitle: string;
+    experiments: DotExperiment[];
+    experimentsFilterStatus: Array<string>;
+    status: LoadingState;
 }
 
 const initialState: DotExperimentsState = {
     pageId: '',
-    state: LoadingState.INIT
+    pageTitle: '',
+    experiments: [],
+    experimentsFilterStatus: [
```

Comment:
```suggestion
    filterStatus: [
```

We are already in "experiment"